src/templates/customer/standing_orders/update_ship_addr.template.html

```diff
@@ -115,101 +103,94 @@
+						<td><select type="text" id="ship_address_sel">
+								<option value=""></option>
 <!--##[%address_book%]
 [%PARAM *body%]##-->
 <option value="[@id@]" [%DATA id:'id' if:'eq' value:'[@address_book@]'%]selected[%END DATA%]>[@ship_title@] ([@ship_zip@])</option>
-<!--##[%END PARAM%] 
+<!--##[%END PARAM%]
 [%END address_book%]##--> </select> <input type="button" value="Select" onClick="chaddr()"></td>
-          </tr>
-        </table>
-        <table cellpadding="4" cellspacing="0" border="0">
-          <tr>
-            <td><b>First Name</b></td>
-            <td><input name="ship_first_name" size="50" maxlength="50" value="[@ship_first_name@]"></td>
-          </tr>
-          <tr>
-            <td><b>Last Name</b></td>
-            <td><input name="ship_last_name" size="50" maxlength="50" value="[@ship_last_name@]"></td>
-          </tr>
-          <tr>
-            
-      <td><strong>Company</strong></td>
-            <td><input name="ship_company" size="50" maxlength="50" value="[@ship_company@]"></td>
-          </tr>
-          <tr>
-            
-      <td><strong>Address</strong></td>
-            <td><input name="ship_street1" size="50" maxlength="50" value="[@ship_street1@]"></td>
-          </tr>
-          <tr>
-            <td>&nbsp;</td>
-            <td><input name="ship_street2" size="50" maxlength="50" value="[@ship_street2@]"></td>
-          </tr>
-          <tr>
-            
-      <td nowrap><strong>Postal Code</strong></td>
-            <td><input name="ship_zip"  value="[@ship_zip@]" size="50" maxlength="10" onChange="updloca('ship')" onKeyUp="updloca('ship')"></td>
-          </tr>
-          <tr>
-            
-      <td valign="top"><strong>Suburb Selector</strong></td>
-            <td valign="top" nowrap><div id="suburb_diship">&nbsp;</div>
-              <div style="display:none;" id="suburb_mgship">&nbsp;</div></td>
-          </tr>
-          <tr>
-            
-      <td><strong>City</strong></td>
-            <td><input type="text" name="ship_city" readonly="1" style="color:#666666;" value="[@ship_city@]" size="50" maxlength="50"></td>
-          </tr>
-          <tr>
-            
-      <td><strong>State</strong></td>
-            <td><input type="text" name="ship_state" readonly="1" style="color:#666666;" value="[@ship_state@]" size="50" maxlength="50"></td>
-          </tr>
-          <tr>
-            
-      <td><strong>Country</strong></td>
-            <td><select name="ship_country" size="1" onChange="updloca('ship')">
+					</tr>
+				</table>
+				<table cellpadding="4" cellspacing="0" border="0">
+					<tr>
+						<td><b>First Name</b></td>
+						<td><input name="ship_first_name" size="50" maxlength="50" value="[@ship_first_name@]"></td>
+					</tr>
+					<tr>
+						<td><b>Last Name</b></td>
+						<td><input name="ship_last_name" size="50" maxlength="50" value="[@ship_last_name@]"></td>
+					</tr>
+					<tr>
+			<td><strong>Company</strong></td>
```

Comment:
The spaces have been replaced with tabs but the indenting is still off around here